### DISCUSSION TITLE
TemplateSnippet.md

### DISCUSSION THREAD

**Discussion by fade2gray** [mark-wiemer/ahkpp#495] [General]
Ref: [TemplateSnippet.md](https://github.com/mark-wiemer-org/ahkpp/blob/main/docs/TemplateSnippet.md) documentation.

Entering "snippets" in the Command Palette, gives the option to "Configure Snippets", not "Configure User Snippets".

Selecting "Configure Snippets", gives the option to edit either the old `"ahk.json (AutoHotkey v1)"` or `"ahk2.json (AutoHotkey v1)"`, which are in `"C:\Users\UserName\AppData\Roaming\Code\User\snippets\"`, but the current v1 and v2 template snippets, outlined at the bottom of the document, are in
`"C:\Users\UserName\.vscode\extensions\mark-wiemer.vscode-autohotkey-plus-plus-6.0.0\language\snippetsV1.json"`
and
`"C:\Users\UserName\.vscode\extensions\mark-wiemer.vscode-autohotkey-plus-plus-6.0.0\language\snippetsV2.json"`

`snippetsV1.json` and `snippetsV2.json` can not be accessed from the Command Palette. If you are new to AHK++, I guess ahk.json and ahk2.json won't be seen in the CP, but surely the new snippetsV1.json and snippetsV2.json should be?

**Comment by mark-wiemer** [comment]
Ref #498, this is a bug, sorry about that
